README.md

```diff
@@ -371,6 +380,15 @@ var objectElement = new minim.ObjectElement({ foo: 'bar' });
 var value = objectElement.get('foo') // returns string instance for 'bar'
 ```
 
+##### getValue
+
+The `getValue` method returns the value of the `ObjectElement` instance at the given name.
+
+```javascript
+var objectElement = new minim.ObjectElement({ foo: 'bar' });
+var value = objectElement.getValue('foo') // returns 'bar'
```

Comment:
Added an issues to discuss this.

https://github.com/refractproject/minim/issues/49